nilearn/input_data/multi_nifti_masker.py

```diff
@@ -267,16 +270,19 @@ def transform_imgs(self, imgs_list, confounds=None, copy=True, n_jobs=1):
                                     'copy'])
 
         func = self._cache(filter_and_mask,
-                          ignore=['verbose', 'memory', 'memory_level', 'copy'])
+                           ignore=['verbose', 'memory', 'memory_level',
+                                   'copy'],
+                           shelve=self._shelving)
         data = Parallel(n_jobs=n_jobs)(
             delayed(func)(imgs, self.mask_img_, params,
-                           memory_level=self.memory_level,
-                           memory=self.memory,
-                           verbose=self.verbose,
-                           confounds=cfs,
-                           copy=copy)
+                          memory_level=self.memory_level,
+                          memory=self.memory,
+                          verbose=self.verbose,
+                          confounds=cfs,
+                          copy=copy,
+                          return_affine=False)
```

Comment:
While I agree that all these cosmetic changes make the code abide better to standards, they make reviewing the PR hard. :(